src/m365/todo/commands/task/task-add.spec.ts

```diff
@@ -62,18 +62,18 @@ describe(commands.TASK_ADD, () => {
       }
     };
     (command as any).items = [];
-    postStub = sinon.stub(request, 'post').callsFake((opts: any) => {
+    postStub = sinon.stub(request, 'post').callsFake(async (opts: any) => {
       if (opts.url === `https://graph.microsoft.com/v1.0/me/todo/lists/AQMkADlhMTRkOGEzLWQ1M2QtNGVkNS04NjdmLWU0NzJhMjZmZWNmMwAuAAADKvwNgAMNPE_zFNRJXVrU1wEAhHKQZHItDEOVCn8U3xuA2AABmQeVPwAAAA==/tasks`) {
-        return Promise.resolve(postRequestData);
+        return postRequestData;
       }
-      return Promise.reject();
+      throw null;
```

Comment:
Shall we throw an 'invalid request' here? Just like we do elsewhere. The code is not meant to reach this point.

Also do that for all other instances of throw null